qreu/email.py

```diff
@@ -177,6 +178,8 @@ def add_header(self, header, value):
         """
         if not (header and value):
             raise ValueError('Header not provided!')
+        if header.lower() == 'date':
```

Comment:
It should check for any existing headers, to prevent more than one header is added.
OR improve replacing the existing header

Reply:
I'm leaving this on an Issue, as it shouldn't happen often and can be a future improvement.

Written on #22